tests/functional/labels-auth.js

```diff
@@ -39,6 +40,58 @@ define(
             .click()
             .end()
             .findByCssSelector(".js-LabelEditor")
+            .end()
+            .findByCssSelector(".js-LabelEditorLauncher")
+            .click()
+            .getAttribute("class")
+            .then(function(className) {
+              assert.notInclude("is-active", className);
+            })
+            .end();
+        },
+
+        "Label editor opens then closes (key events)": function() {
+          return FunctionalHelpers.openPage(
+            this,
+            url("/issues/2"),
+            ".js-LabelEditorLauncher",
+            true /* longerTimeout */
+          )
+            .findByCssSelector("body")
+            .type("l")
+            .end()
+            .findByCssSelector(".wc-LabelEditor-search")
+            .type(keys.ESCAPE)
```

Comment:
random weird fact, firefox will pass with `type(keys.ESC)` but Chrome will not. (`keys.ESCAPE` is what is documented)

:man_shrugging: :man_shrugging: :man_shrugging: